akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,17 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.loggers.pinned-dispatcher"
```

Comment:
@patriknw

Hmmm, too many threads? I understand what you are saying but ... it is an I/O thread, I wanted to make it use `akka.io.pinned-dispatcher` but then I would be messing with the selector functionality/thread whatever that is ;-)

I can change it to default `akka.io.pinned-dispatcher` just let me know, I would just force push.